frontend/components/Avatar/Avatar.tsx

```diff
@@ -14,61 +14,42 @@ interface IFleetAvatarProps {
 const FleetAvatar = ({ className }: IFleetAvatarProps) => {
   return (
     <svg
-      data-testid="fleet-avatar"
-      className={className}
```

Comment:
Looks like removing this prop may break some styles downstream, e.g. instances of `Avatar` that use `hasWhiteBackground`. If the intention is to remove all of those styles we probably want to clean up all of that logic as well?